apps/web/src/app/arrangementer/components/EventFilters.tsx

```diff
@@ -1,31 +1,210 @@
+      },
+      values.viewMode
+    )
   }
 
   return (
-    <form onSubmit={form.handleSubmit(handleSubmit)}>
-      <TextInput placeholder="Søk etter arrangementer..." {...form.register("bySearchTerm")} />
-    </form>
+    <div className="border-gray-200 dark:border-stone-700 h-fit rounded-lg border shadow-b-sm py-4">
+      <div className="mx-4">
+        <form onSubmit={form.handleSubmit(handleSubmit)} className="flex flex-col gap-4">
+          <div>
+            <Label htmlFor="bySearchTerm" className="mb-1 text-md">
+              Søk
+            </Label>
+            <TextInput placeholder="Søk etter arrangementer..." {...form.register("bySearchTerm")} id="bySearchTerm" />
+          </div>
+          <Controller
+            control={form.control}
+            name="byType"
+            render={({ field: { onChange, value } }) => (
+              <div>
+                <Label htmlFor="byType" className="mb-1 text-md">
+                  Type
+                </Label>
+                <Select value={value} onValueChange={onChange}>
+                  <SelectTrigger id="byType">
+                    <SelectValue />
+                  </SelectTrigger>
+
+                  <SelectContent>
+                    <SelectGroup>
+                      <SelectItem value="ALL" key="ALL">
+                        Alle
+                      </SelectItem>
+                      {EVENT_TYPE_OPTIONS.map((type) => (
+                        <SelectItem value={type.value} key={type.value}>
+                          {type.label}
+                        </SelectItem>
+                      ))}
+                    </SelectGroup>
+                  </SelectContent>
+                </Select>
+              </div>
+            )}
+          />
+
+          <Controller
+            control={form.control}
+            name="byOrganizingGroup"
+            render={({ field: { onChange, value } }) => (
+              <div>
+                <Label htmlFor="byOrganizingGroup" className="mb-1 text-md">
+                  Arrangør
+                </Label>
+                <GroupSelect onChange={onChange} groups={groups} value={value} />
+              </div>
+            )}
+          />
+
+          <Controller
+            control={form.control}
+            name="viewMode"
+            render={({ field: { onChange, value } }) => (
+              <div>
+                <Label htmlFor="viewMode" className="mb-1 text-md">
+                  Sorter
+                </Label>
+                <Select value={value} onValueChange={onChange}>
+                  <SelectTrigger id="viewMode">
+                    <SelectValue />
+                  </SelectTrigger>
+
+                  <SelectContent>
+                    <SelectGroup>
+                      <SelectItem value="BY_CATEGORY">Kategori</SelectItem>
+                      <SelectItem value="BY_DATE">Dato</SelectItem>
```

Comment:
I would maybe name these "attendance" (påmelding) and "chronological" (kronologisk)